src/pages/audio-rewards-page/components/modals/ChallengeRewards.tsx

```diff
@@ -293,6 +294,9 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
     if (claimStatus === ClaimStatus.SUCCESS) {
       toast(messages.rewardClaimed, CLAIM_REWARD_TOAST_TIMEOUT_MILLIS)
     }
+    if (claimStatus === ClaimStatus.ALREADY_CLAIMED) {
+      toast(messages.rewardAlreadyClaimed, CLAIM_REWARD_TOAST_TIMEOUT_MILLIS)
+    }
```

Comment:
Thanks for handling this one - was tracking it on linear as part of another issue but it slipped through the cracks. The way Saliou and I were discussing handling this was to just treat this the same as success case, but since we're polling for balance updates now, it would be odd if your balance had already increased and doesn't increase with the claim, so I think this solution makes sense.